test/__tests__/data/customerio_output.json

```diff
@@ -1141,5 +1141,167 @@
     "files": {},
     "userId": "e91e0378-63fe-11ec-82ac-0a028ee659c3",
     "statusCode": 200
+  },
+  {
+    "version": "1",
+    "type": "REST",
+    "method": "POST",
+    "endpoint": "https://track.customer.io/api/v2/batch",
+    "headers": {
+      "Authorization": "Basic YzBlZmRiZDIwYjlmYmUyNGE3ZTI6ZWYzMmMzZjYwZmI5OGYzOWVmMzU="
+    },
+    "params": {},
+    "body": {
+      "XML": {},
+      "FORM": {},
+      "JSON": {
+        "type": "object",
+        "action": "identify",
```

Comment:
The documentation for the [v2 batch endpoint](https://www.customer.io/docs/api/track/#operation/batch) indicates that there should be an object with a top-level `batch` array here, rather than just specifying the object directly.

Reply:
@orrc we are handling that structure at router level, you can check customerio_router_output.json test cases